packages/trader/src/Modules/Contract/Components/Sell/sell-button.tsx

```diff
@@ -1,10 +1,16 @@
-import { observer } from 'mobx-react';
-import PropTypes from 'prop-types';
 import React from 'react';
 import { Button, Popover } from '@deriv/components';
+import { TContractInfo } from '@deriv/shared';
 import { localize } from '@deriv/translations';
 
-const SellButton = ({ contract_info, is_sell_requested, is_valid_to_sell, onClickSell }) => {
+type TSellButton = {
+    contract_info: TContractInfo;
+    is_sell_requested: boolean;
+    is_valid_to_sell: boolean;
+    onClickSell: () => void;
+};
+
+const SellButton = ({ contract_info, is_sell_requested, is_valid_to_sell, onClickSell }: TSellButton) => {
```

Comment:
We don't use `SellButton` component in project anymore (at least I wasn't able to find it), but I didn't delete it as this situation could change.